src/ViewModels/ArcGisMapServerItemViewModel.js

```diff
@@ -43,7 +43,7 @@ defineProperties(ArcGisMapServerItemViewModel.prototype, {
      */
     type : {
         get : function() {
-            return 'esri-mapServer';
+            return 'esri-rest';
```

Comment:
Why this change?  Esri has lots of REST APIs.  `esri-mapServer` is a much more precise description of what this catalog item talks to.